src/ui/components/search/autocompletecomponent.js

```diff
@@ -358,6 +379,28 @@ export default class AutoCompleteComponent extends Component {
     this._isVoiceSearchActive = value;
   }
 
+  setCustomPrompts (prompts) {
+    this.customPrompts = prompts;
+  }
+
+  createSectionForCustomPrompts (customPrompts) {
+    const customAutocompleteResults = customPrompts
+      .map(prompt => AutoCompleteResponseTransformer._transformAutoCompleteResult({ value: prompt }));
```

Comment:
Could we remove the underscore from `_transformAutoCompleteResult` if it's being used "publicly" now?

Reply:
sure!